src/log/zipkin.cpp

```diff
@@ -155,13 +168,35 @@ fc::variant create_zipkin_variant( zipkin_span::span_data&& span, const std::str
    return result;
 }
 
+void zipkin::post_request(zipkin_span::span_data&& span) {
+   boost::asio::post(my->work_strand, [this, span{std::move(span)}]() mutable {
+         my->log( std::move( span ) );
+   });
+}
+
 void zipkin::log( zipkin_span::span_data&& span ) {
-   if( my->consecutive_errors > my->max_consecutive_errors || my->stopped )
+   if( my->stopped ) {
       return;
+   }else if( fc::sighup_requested.load()) {
+      fc::sighup_requested = false;
+      my->consecutive_errors = 0;
+   }else if( my->consecutive_errors > my->max_consecutive_errors ) {
+      return;
+   }
 
-   boost::asio::post(my->work_strand, [this, span{std::move(span)}]() mutable {
-      my->log( std::move( span ) );
-   });
+   if( my->consecutive_errors > 0 ) {
+      if( my->timer_expired ) {
+         my->timer_expired = false;
+         my->timer.expires_from_now(boost::posix_time::microsec(my->retry_interval_us));
+         my->timer.async_wait([this, span{std::move(span)}](auto&) mutable {
+            ilog("Retry connecting to zipkin: ${u} ...", ("u", my->zipkin_url) );
```

Comment:
I think this log statement should be moved into the `else if( fc::sighup_requested.load()) {` above.

Reply:
Fixed.